x-pack/plugins/ml/public/application/services/job_cloning_service.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isPopulatedObject } from '@kbn/ml-is-populated-object';
+import type { JobCreator } from '../jobs/new_job/common/job_creator/job_creator';
+
+interface TempJobCloningObjects {
```

Comment:
Nit: Wondering why this is plural and not singular `TempJobCloningObject`? Maybe replace with `TempJobCloningData`?

Reply:
Updated in https://github.com/elastic/kibana/pull/191724/commits/4a13c21955df48db93309c6b24ff73ddd3765ff7